Makefile

```diff
@@ -71,11 +71,11 @@ get_vendor_deps: ensure_tools
 
 metalinter:
 	protoc $(INCLUDE) --lint_out=. types/*.proto
-	gometalinter --vendor --deadline=600s --enable-all --disable=lll ./...
+	gometalinter.v2 --vendor --deadline=600s --enable-all --disable=lll ./...
```

Comment:
we don't need the same `$(GOPATH)/bin/` here?

Reply:
yea, not sure why. Suspect it has to do with the circle setup

Reply:
ok